package.json

```diff
@@ -11,12 +11,12 @@
   "main": "server.js",
   "dependencies": {
     "async": "*",
-    "americano": "=0.3.3",
-    "americano-cozy": "=0.2.2",
-    "cozy-realtime-adapter": "=0.12.0"
+    "americano": "=0.4.2",
```

Comment:
To keep the Cozy repositories consistent, the guideline to write dependency version is to not use the `=`.  It has the same effect and unclutters a little bit the manifest.